src/hooks/useProfileEditNewPassword.ts

```diff
@@ -58,11 +58,11 @@ const useProfileEditNewPassword = () => {
                     position: 'bottom',
                     padding: '1rem',
                     customClass: {
```

Comment:
Does it make sense to have a popupClass for all of popups? Then you simply will need to do something like `customClass:  popupClass` or something like that.

Reply:
Thanks for the suggestion! I have implemented a solution to handle all of our popups with a single component called AppPopupAlert. This component takes in props such as title, icon, text, ConfirmButtonText, and CancelButtonText, which allows us to easily customize the appearance and behavior of our alerts without having to repeat the same code in multiple places. Let me know if you think this solution is okay and if there are any further improvements that can be made. 😄